private-templates-service/adaptivecards-templating-service/src/models/models.ts

```diff
@@ -0,0 +1,27 @@
+export interface IUser {
+    id?: string;
+    team: string[];
+    org: string[];
+    email: string;
+  }
+  
+export interface ITemplateInstance {
+    json: string;
+    version: string;
+}
+
+export interface ITemplate {
+    id?: string;
+    instances: ITemplateInstance[];
+    tags: string[];
+    owner: string;
+    createdAt?: Date;
+    updatedAt?: Date;
+    isPublished?: boolean;
+}
+
+export interface JSONResponse<T> {
+    success: boolean;
+    errorMessage?: string;
+    result?: T;
+}
```

Comment:
Can these be exported again in an `index.ts` file

Reply:
Fixed!